lib/entities/organization.ts

```diff
@@ -1,27 +1,21 @@
 import cloneDeep from 'lodash/cloneDeep'
 import { freezeSys, toPlainObject } from 'contentful-sdk-core'
 import enhanceWithMethods from '../enhance-with-methods'
+// @ts-ignore
```

Comment:
This is temporary. As soon as we convert create-org-api into ts we would be able to remove it? WDYT about this solution? or maybe it would be better to do it some other way?

Reply:
Let's use ``// @ts-expect-error`` and remove changes `eslint` file

Reply:
For some reason it does not work when it comes to imports 😕 that's why I used ignore here. But it's possible that I do something wrong.

Reply:
Then

```
// eslint-disable-next-line
// @ts-ignore
```

without making changes in .eslint file

Reply:
@suevalov ✅